foreign_cc/private/framework/toolchain.bzl

```diff
@@ -72,9 +66,8 @@ framework_toolchain_repository = repository_rule(
     doc = "A repository rule which defines a `@rules_foreign_cc//foreign_cc/private/framework:shell_toolchain` toolchain.",
     implementation = _framework_toolchain_repository_impl,
     attrs = {
-        "commands_src": attr.label(
-            doc = "The label of a `.bzl` source which defines toolchain commands",
-            allow_files = [".bzl"],
+        "commands_src": attr.string(
+            doc = "The string of a `.bzl` source which defines toolchain commands",
```

Comment:
Why remove the `.bzl` restriction?

Reply:
because it becomes a string here - bzlmod remaps the names of the repos and so we can't evaluate the Label at this point as it gets used in  the context of another repo so unfortunately we need to pass it through as a string here. It's a bit brittle but as its all internal it works but isn't great.
I guess the real solution is to move the `.bzl` into the generated repo itself so we don't need to resolve a label to access it?

Reply:
Oooh, I didn't notice that the type changed. I'm more nervous about making larger changes than needing to restrict the values here. This seems fine.